tests/test_flip.py

```diff
@@ -650,7 +650,6 @@ def test_should_replace_pending_transactions_if_model_lowers_bid_and_increases_g
         # and
         simulate_model_output(model=model, price=Wad.from_number(15.0), gas_price=15)
         # and
-        self.keeper.check_all_auctions()
```

Comment:
This call delays execution of the test long enough that the asynchronously-submitted TX completes before it can be replaced.  It isn't necessary for execution of this particular test, and is omitted in similar tests.